api/chalicelib/core/tags.py

```diff
@@ -0,0 +1,49 @@
+import schemas
+from chalicelib.utils import helper
+from chalicelib.utils import pg_client
+
+
+def create_tag(project_id: int, data: schemas.TagCreate) -> int:
+    query = """
```

Comment:
I think name is missing

Reply:
Fixed in https://github.com/openreplay/openreplay/pull/1834/commits/4fd84ae32fce0b69019b31df9b81d999b22a4c4f